sc/social/like/plugins/facebook/browser.py

```diff
@@ -108,6 +104,46 @@ def typebutton(self):
             self.width = '55px'
```

Comment:
why not use ``button_count`` and ``box_count`` as value of the typebutton controlpanel option?